sdk/dotnet/AccessPackageAssignmentPolicy.cs

```diff
@@ -30,26 +30,26 @@ namespace Pulumi.AzureAD
     /// 
     /// return await Deployment.RunAsync(() =&gt; 
     /// {
-    ///     var exampleGroup = new AzureAD.Group("exampleGroup", new()
+    ///     var example = new AzureAD.Group("example", new()
```

Comment:
Are these intentional? Seems like all the resources here have the same name now.

Reply:
Yeah we discussed it with platform team, it'd be nice to have it the other way but not blocking on it.